main.go

```diff
@@ -495,10 +496,99 @@ func employeeDashboard(w http.ResponseWriter, r *http.Request) {
 
 	if val.(*LogInSessionCookie).ProfileType == "employee" {
 		// TODO: Pass in the correct name that is stored in cookies
-		RenderTemplate(w, "employeehomescreen.html", pongo2.Context{"fname": "Alex"})
+		RenderTemplate(w, "employeehomescreen.html", pongo2.Context{"fname": "Alex", "flashes": RetrieveFlashes(r, w)})
 	} else {
 		// TODO: redirect to user accounts page instead
 		// http.Redirect(w, r, "/accounts", http.StatusSeeOther)
-		RenderTemplate(w, "employeehomescreen.html", pongo2.Context{"fname": "Alex"})
+		RenderTemplate(w, "employeehomescreen.html", pongo2.Context{"fname": "Alex", "flashes": RetrieveFlashes(r, w)})
 	}
 }
+
+func addUser(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodPost {
+		// Parse form data
+		err := r.ParseForm()
+		if err != nil {
+			http.Error(w, "Unable to parse form data", http.StatusBadRequest)
+			return
+		}
+
+		// Extract the data from the form
+		firstName := r.FormValue("fname")
+		lastName := r.FormValue("lname")
+		email := r.FormValue("email")
+		phoneNum := r.FormValue("phonenum")
+		carrier := r.FormValue("carrier")
+		password := r.FormValue("pw")
+		dob := r.FormValue("dob") // Date of Birth
+		billingAddress := r.FormValue("billing_address") // Billing Address
+
+		// Hash the password
+		passwordHash, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
+		if err != nil {
+			http.Error(w, "Failed to hash password", http.StatusInternalServerError)
+			return
+		}
+
+		// Insert the new user into the database
+		err = OpenDBConnection(func(conn *pgxpool.Pool) error {
+			_, err := conn.Exec(
+				context.Background(),
+				`INSERT INTO profiles (
+					profile_type,
+					first_name,
+					last_name,
+					email,
+					date_of_birth,
+					billing_address,
+					phone_number,
+					phone_carrier,
+					password_hash
+				) VALUES (
+					'customer', 
+					COALESCE(NULLIF($1, ''), 'Admin'), 
+					COALESCE(NULLIF($2, ''), 'User'), 
+					COALESCE(NULLIF($3, ''), 'admin@company.com'), 
```

Comment:
I think when creating a customer we don't need default values, they should be required